### DISCUSSION TITLE
Layout primary navigation

### DISCUSSION THREAD

**Discussion by johannadm** [netwerk-digitaal-erfgoed/solid-crs#93] [Collections]
Hi all,

We would like to hear your opinion about the primary navigation that will be used in the application. The content of the collections page will be discussed in another discussion. So in here, let's focus only on the blue navigation bar on the left side of the screen.

The navigation bar exists out of 3 sections: the logo and log out button on the top, the search bar in the middle, and the collections below. 
![afbeelding](https://user-images.githubusercontent.com/80322390/116694212-48ca5780-a9bf-11eb-8bd1-2ff054c7875a.png)

When a user logs in for the first time, there will always be an empty collection to start with. There is always one collection needed that can't be erased in the navigation. 
![afbeelding](https://user-images.githubusercontent.com/80322390/116693490-49162300-a9be-11eb-8b8f-0275ed3ae828.png)

When a user performs a search task, the results will show up in the primary navigation bar. The collections will be hidden untill the searching is dismissed.
![afbeelding](https://user-images.githubusercontent.com/80322390/116695429-e8d4b080-a9c0-11eb-948d-52777c0605d4.png)

Let us know what you think!

**Comment by ddeboer** [comment]
@sdevalk and I have some reservations about showing the search results in the primary navigation. 

1. Collection objects may not be identifiable by name only, for example when many objects have the same name. To include attributes needed for identification, including the object’s image, space in the primary navigation is insufficient.
2. It will be hard to navigate large amounts of search results.
3. The search results are hard to extend later, for example when adding faceted search (out of scope), which needs more screen estate.

Therefore, we suggest showing the search results in the [content pane](https://github.com/digita-ai/nde-erfgoedinstellingen/discussions/94) instead. I _think_ the search input can remain where it is, but perhaps it needs to move to the right as well.

Before we reach a final decision, could you please briefly explain your reasons for having the search results in the sidebar?

@johannadm Feel free to move this comment to a separate discussion about search results if you think that’s better.

**Reply by ddeboer** [reply]
See https://github.com/digita-ai/nde-erfgoedinstellingen/discussions/143 for the new search design proposal.
